controllers/operatorchecker/operatorchecker_controller.go

```diff
@@ -114,7 +119,9 @@ func (r *Reconciler) getCSVBySubscription(ctx context.Context, subscriptionInsta
 
 	var matchCSVList = []olmv1alpha1.ClusterServiceVersion{}
 	for _, csv := range csvList.Items {
-		if strings.Contains(csv.Name, subscriptionInstance.Name) {
+		properties := csv.GetAnnotations()["operatorframework.io/properties"]
+		csvPackageName := strings.SplitN(strings.SplitN(properties, "packageName", 2)[1][3:], "\"",2)[0]
```

Comment:
`properties` is a string that cannot be converted to a map.

Reply:
Could we use `strings.Contains` here?

Reply:
good idea!